ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8304,3 +8308,19 @@ DTPM:
   - detergent_pump_run_command
   implements:
   - MONITORING
+
+APM:
```

Comment:
can the existing `ZSPM` type be used instead?

Reply:
updated